# La Commune de Paris 1871 en liberté
ou La Commune dans Wikidata

Ceci est un site personnel qui utilise des données issues de l'univers wikimédia dont wikipédia, commons, wikidata, wikisources...

Table des matières

- [Présentation](#Présentation)
    - [Point grammatical](#Point_grammatical)
    - [Qui suis-je pour le faire ?](#Quisuis-jepourlefaire?)
- [Méthodologie](#Méthodologie)
    - [Wikidata : élément, propriété, valeurs.](#Wikidata:élément,propriété,valeurs.)
- [Technique](#Technique)
- [Faire une requête sur wikidata en python](#Faireunerequêtesurwikidataenpython)
    - Méthode
- [Portraits de Communardes et Communards](#PortraitsdeCommunardesetCommunards)
    - [Trouver des communard·e·s sur wikidata](#Trouverdescommunard·e·ssurwikidata)
        - L'élément "communard ou communarde"
        - La propriété "occupation"
        - Le triplet ?communard wdt:P106 wd:Q1780490.
    - Les communard·e·s connu·e·s
        - Liste complète
        - Liste complète avec Reasonator
        - Nombre
        - Quelques tests possibles pour vérifier la fiabilité des données.
            - Pour vérifier qu'ils ne sont pas née après ou mort avant la Commune
            - Des êtres vivants
            - Quelques sources
    - Présentations et statistiques à propos des Communard-e-s
        - Galerie de portrait
        - Gender-gap 
        - A part avoir participer à la commune, que faisaient ces personnes ?
            - Au moins deux personnes qui l'occupe
        - Quel age avait ces personnes en 1871
     - Relations entre communard·e·s
        - Généalogie
        - Reseau social des communard·e·s sur wikidata
            - Graph des communard·e·s et des organismes dont als sont membres
            - nombre par organisme
- [Les communardes et communards dans des cimetières](#Lescommunardesetcommunardsdansdescimetières)
     - Généralités
     - Nombre de personne par cimetière
     - Cimetière du Montparnasse
     - Cimetière du Père la Chaise
- Une chronologie
- Les affiches
- Elle n'est pas morte

## TODO

- [ ] attention, langue auto est pluto en GB que fr, pas mal de mots sortent en anglais
- [ ] point médian au lieu de -
- [ ] attribuer une licence

## Présentation

### Point grammatical
Pour un certains nombre de raison, j'use du genre grammatical neutre, en l'écrivant selon les règles de grammaire proposées par Alphératz dans son ouvrage Grammaire inclusive du Français. Je signale la première occurrence de ces mots par un \*.

J'emploie ces tournures pour parler de moi-même, et pour éviter des doubles flexions comme communardes et communards.

En neutre, al* faudrait écrire an Communarx et des Communarz. Pour que les moteurs de recherche puissent trouver le site, je décide de garder selon les moments communardes et communards ou ça forme raccourcie communard·e·s.

### Qui suis-je pour le faire ?
Il se trouve que je suis membre de l'association les Amies et amis de la Commune de Paris 1871 depuis pas mal de temps (plus d'une décennie, c'est sûr) quand je n'oublie pas de cotiser... Toutefois, ce site est fait de façon autonome de l’association.

Je ne suis pas du tout historian*, mais geekette, c'est à dire passionæ* d'informatique.
J'ai commenæ à contribuer sur Wikipédia en 2006 puis sur Wikidata à partir de 2017. J'y contribue sous le pseudo Silanoc.

Je ne sais pas aller chercher l'information dans les sources primaires de l'époque, dans les archives... c'est du ressort des histoirans. Par contre j'aime rendre les informations, la donnée, la data accessibles au plus grand nombre. Et c'est là que Wikidata intervient.

C'est un projet libre et gratuit. Tout le monde peut y accéder, y contribuer. N'étant pas rédigé comme Wikipédia, mais étant une base de données, le système a été conçu comme naturellement multilinguistique. De ce fait, toutes les modifications sont accessibles par les locutaires* du monde entier (à quelques détails près). Et c'est très utilisé pour tout ce qui touche à "l'intelligence artificielle" car elle a besoin d'une base de données. Autant lui donner des informations sur un sujet trop souvent ignoré.

> Wikidata est une base de connaissances libre et gratuite qui peut être lue et modifiée tant par des personnes que par des dispositifs informatisés. 
page d'acceuil de wikipédia</quote>

Tout ce site/article est basé sur des données de Wikidata ou des images de Commons. Cela par définition sous licence libre (plusieurs existes). Mes propos sont également sous licence créative commons.

## Méthodologie
Mon objectif est de travailler qu'avec des données contenues dans l'univers wikimédia : wikidata, commons, wikipédia, éventuellement wikisources, wikiquote...

Les données sont collaboratives. Cela nécessite de faire confiance au travail des autres sur lequel je m'appuie. Le maximum d'information doit être sourcé. Dans l'analyse statistique/globale, je fais le pari de leur fiabilité. En cas de doute, il est facile de vérifier au cas par cas.

Les données sont celles que les autres personnes ont déjà rentrées. Il y a donc des trous, des blancs, des manques.

Chacun de mes résultats est donc à introduire par : "en fonction du contenu de wikidata...".

Ce n'est pas une vérité totale et absolue. Mais une partie de la vérité.

C'est un travail vérifiable et reproductible. Toutes les données sont publiques (par définition) et mes méthodes de recherche/calcul sont communicables. Aussi, toute personne peut les évaluer et refaire.

C'est un contenu évolutif en fonction des apports/modifications/suppressions fait à la base de données.

J'apporte aussi du contenu dedans, par des photos, des ajouts, des précisions en fonction de ce que je peux remplir sans me tromper.

### Wikidata : élément, propriété, valeurs.
Wikidata stocke des éléments. Tout ce qui existe (au sens qui peut être décrit) est un potentiel élément. Aussi chaque communard·e·s, barricades, livres, événements... sont un élément. Ils commencent par la lettre Q, puis un nombre en fonction de son ordre de création.

Chaque élément possède une ou plusieurs propriétés. Cela peut être un prénom, une date de création, un coordonnées géographique... Ils commence par un P.

À chaque propriété d'un élément il existe une ou plusieurs valeurs.Cela peut être un nombre, une date, ou un autre élément de wikidata.

Cela fait une série de triplets : éléments, propriété, valeurs.

Plus de détails : https://www.wikidata.org/wiki/Wikidata:Introduction/fr

## Technique

In [1]:
# les installations nécessaires
# https://rdflib.github.io/sparqlwrapper/
!pip install sparqlwrapper

Defaulting to user installation because normal site-packages is not writeable


## Faire une requête sur wikidata en python

### Méthode

La classe Extraction_wikidata, est une adaption du modèle de réquête en python qui se trouve sur https://query.wikidata.org/

- dans la variable query, il faut copier/coller la requête faite dans query.wikidata.org
- exécuter les cellules "## 0" à "## 5" comme si dessous afin d'obtenir le résultat dans un dataframe exploitable pour faire des manipulations de données.

In [77]:
## 0 - import

import sys
from SPARQLWrapper import SPARQLWrapper, JSON
import pandas as pd

In [78]:
## 1 - les variables

endpoint_url = "https://query.wikidata.org/sparql"

query = """SELECT ?communard_ou_communarde ?communard_ou_communardeLabel WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
}"""

In [79]:
## 2 - l'objet

class Extraction_wikidata():
    def __init__(self, endpoint_url, query):
        self.endpoint_url = endpoint_url
        self.query = query
        
    
    def get_results(self, endpoint_url, query):
        user_agent = "WDQS-example Python/%s.%s" % (sys.version_info[0], sys.version_info[1])
        # TODO adjust user agent; see https://w.wiki/CX6
        sparql = SPARQLWrapper(endpoint_url, agent = user_agent)
        sparql.setQuery(query)
        sparql.setReturnFormat(JSON)
        return sparql.query().convert()
    
    def json_to_df(self, entree):
        #results = sparql.query().convert()
        df_results = pd.json_normalize(entree["results"]["bindings"])
        return df_results
    
    def extraire_et_df(self):
        extract_json = self.get_results(self.endpoint_url, self.query)
        extract_df = self.json_to_df(extract_json)
        return extract_df


In [80]:
## 3 - instantiation de l'objet et récupération du df
extraction_communard = Extraction_wikidata(endpoint_url, query)
df_communard = extraction_communard.extraire_et_df()

In [81]:
## 4 - afficher le df voulu
print(df_communard)

    communard_ou_communarde.type              communard_ou_communarde.value   
0                            uri      http://www.wikidata.org/entity/Q20951  \
1                            uri      http://www.wikidata.org/entity/Q34618   
2                            uri     http://www.wikidata.org/entity/Q184535   
3                            uri     http://www.wikidata.org/entity/Q216092   
4                            uri     http://www.wikidata.org/entity/Q257950   
..                           ...                                        ...   
292                          uri  http://www.wikidata.org/entity/Q115257384   
293                          uri  http://www.wikidata.org/entity/Q115257385   
294                          uri  http://www.wikidata.org/entity/Q115257386   
295                          uri  http://www.wikidata.org/entity/Q115257387   
296                          uri  http://www.wikidata.org/entity/Q115257388   

    communard_ou_communardeLabel.xml:lang communard

In [82]:
## 5 - quelques statistiques sur le df
df_communard.describe()

communard_ou_communarde.type          communard_ou_communarde.value   
count                           297                                    297  \
unique                            1                                    297   
top                             uri  http://www.wikidata.org/entity/Q20951   
freq                            297                                      1   

       communard_ou_communardeLabel.xml:lang   
count                                    293  \
unique                                     1   
top                                       en   
freq                                     293   

       communard_ou_communardeLabel.type communard_ou_communardeLabel.value  
count                                297                                297  
unique                                 1                                297  
top                              literal                      Élisée Reclus  
freq                                 297                                  1

### Avoir TOUTES les informations sur les communard·e·s

pour toutes les personnes, chercher les ?p ?q avec gestion des label.
Attention : c'est lourd !

In [109]:
endpoint_url = "https://query.wikidata.org/sparql"

query = """SELECT ?communard_ou_communarde ?communard_ou_communardeLabel ?p ?pLabel ?q ?qLabel WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "fr,en".
                         ?communard_ou_communarde rdfs:label ?communard_ou_communardeLabel.
                         ?p rdfs:label ?pLabel.
                         ?q rdfs:label ?qLabel.}
  
  ?communard_ou_communarde wdt:P106 wd:Q1780490;
                           ?p ?q. 
}
"""

In [110]:
extraction_communard_tous_p_q = Extraction_wikidata(endpoint_url, query)
df_communard_tous_p_q = extraction_communard_tous_p_q.extraire_et_df()

In [111]:
df_communard_tous_p_q

communard_ou_communarde.type             communard_ou_communarde.value   
0                              uri  http://www.wikidata.org/entity/Q98342898  \
1                              uri  http://www.wikidata.org/entity/Q98342898   
2                              uri  http://www.wikidata.org/entity/Q98342898   
3                              uri  http://www.wikidata.org/entity/Q98342898   
4                              uri  http://www.wikidata.org/entity/Q98342898   
...                            ...                                       ...   
32712                          uri   http://www.wikidata.org/entity/Q3059921   
32713                          uri   http://www.wikidata.org/entity/Q3059921   
32714                          uri   http://www.wikidata.org/entity/Q3059921   
32715                          uri   http://www.wikidata.org/entity/Q3059921   
32716                          uri   http://www.wikidata.org/entity/Q3059921   

      p.type                                            p.value   
0        uri                          http://schema.org/version  \
1        uri                     http://schema.org/dateModified   
2        uri                      http://schema.org/description   
3        uri         http://www.w3.org/2000/01/rdf-schema#label   
4        uri         http://www.w3.org/2000/01/rdf-schema#label   
...      ...                                                ...   
32712    uri          http://www.wikidata.org/prop/direct/P2632   
32713    uri          http://www.wikidata.org/prop/direct/P2671   
32714    uri          http://www.wikidata.org/prop/direct/P4724   
32715    uri  http://www.wikidata.org/prop/direct-normalized...   
32716    uri                  http://www.wikidata.org/prop/P463   

                                      q.datatype   q.type   
0       http://www.w3.org/2001/XMLSchema#integer  literal  \
1      http://www.w3.org/2001/XMLSchema#dateTime  literal   
2                                            NaN  literal   
3                                            NaN  literal   
4                                            NaN  literal   
...                                          ...      ...   
32712                                        NaN      uri   
32713                                        NaN  literal   
32714                                        NaN  literal   
32715                                        NaN      uri   
32716                                        NaN      uri   

                                                 q.value   
0                                             1742086615  \
1                                   2022-10-03T13:18:25Z   
2                                             communarde   
3                                    Angelina Courcelles   
4                                    Angelina Courcelles   
...                                                  ...   
32712            http://www.wikidata.org/entity/Q2879078   
32713                                        /g/121yn5xg   
32714                                              60250   
32715                          http://g.co/kg/g/121yn5xg   
32716  http://www.wikidata.org/entity/statement/Q3059...   

      communard_ou_communardeLabel.xml:lang communard_ou_communardeLabel.type   
0                                        fr                           literal  \
1                                        fr                           literal   
2                                        fr                           literal   
3                                        fr                           literal   
4                                        fr                           literal   
...                                     ...                               ...   
32712                                    fr                           literal   
32713                                    fr                           literal   
32714                                    fr              

In [88]:
df_communard_tous_p_q.describe()

communard_ou_communarde.type          communard_ou_communarde.value   
count                         32717                                  32717  \
unique                            1                                    297   
top                             uri  http://www.wikidata.org/entity/Q34618   
freq                          32717                                   1154   

       p.type                                     p.value q.type   
count   32717                                       32717  32717  \
unique      1                                         909      2   
top       uri  http://www.w3.org/2000/01/rdf-schema#label    uri   
freq    32717                                        5869  16716   

                                    q.value   
count                                 32717  \
unique                                20561   
top     http://www.wikidata.org/entity/Q150   
freq                                    348   

       communard_ou_communardeLabel.xml:lang   
count                                  32717  \
unique                                     1   
top                                       fr   
freq                                   32717   

       communard_ou_communardeLabel.type communard_ou_communardeLabel.value   
count                              32717                              32717  \
unique                                 1                                297   
top                              literal                    Gustave Courbet   
freq                               32717                               1154   

       pLabel.type                                pLabel.value   
count        32717                                       32717  \
unique           1                                         909   
top        literal  http://www.w3.org/2000/01/rdf-schema#label   
freq         32717                                        5869   

       qLabel.xml:lang qLabel.type qLabel.value   
count             5226       32717        32717  \
unique               2           1        20472   
top                 fr     literal     français   
freq              5174       32717          348   

                                      q.datatype q.xml:lang  
count                                       1972      10427  
unique                                         3        203  
top     http://www.w3.org/2001/XMLSchema#integer         fr  
freq                                        1188       1068

In [92]:
df_communard_tous_p_q['pLabel.value']

0         http://www.wikidata.org/prop/direct/P25
1         http://www.wikidata.org/prop/direct/P27
2         http://www.wikidata.org/prop/direct/P31
3         http://www.wikidata.org/prop/direct/P69
4        http://www.wikidata.org/prop/direct/P103
                           ...                   
32712          http://www.wikidata.org/prop/P1559
32713          http://www.wikidata.org/prop/P1596
32714          http://www.wikidata.org/prop/P1871
32715          http://www.wikidata.org/prop/P2671
32716          http://www.wikidata.org/prop/P4724
Name: pLabel.value, Length: 32717, dtype: object

In [106]:
# df_communard_tous_p_q['pLabel.value'] == "http://www.wikidata.org/prop/direct/P25"

## Portraits de Communardes et Communards

### Trouver des communard·e·s sur wikidata

#### L'élément "communard ou communarde"
Sur Wikidata, al existe l'élément communard ou communarde, sous le nombre Q1780490
https://www.wikidata.org/wiki/Q1780490
> communard ou communarde (Q1780490)
personne participant à la Commune de Paris

#### La propriété "occupation"
Sur wikidata, la propriété P106, permet de renseigner l'occupation (au sens très large ) d'une personne.
https://www.wikidata.org/wiki/Property:P106
> occupation (P106)
nature d'activité d'une personne (métier, hobby…) ; voir aussi « domaine d'activité » (P101) et « fonction » (P39)

#### Le triplet ?communard wdt:P106 wd:Q1780490.
Pour toutes les recherche je considère que la personne est communard·e via occupation communard, soit
```?communard wdt:P106 wd:Q1780490.``` si la requête a été écrite avant le 7 janvier 2023 ou `?communard_ou_communarde wdt:P106 wd:Q1780490.` après.

Quand j'ai commencé a m'intéresser à ce sujet c'était ce triplet qui était déjà majoritaire.  Il était en "concurrence" avec  allégeance à Commune, participant à la Commune, conflit commune... Cela aurait nécessité plus de changement.

Pour mémoire il s'agit des triplet 
```
?communard wdt:P945 wd:Q133132.
?communard wdt:P945 wd:Q1249756.
?communard wdt:P463 wd:Q1249756.
?communard wdt:P463 wd:Q133132.
?communard wdt:P607 wd:Q133132.
```

### Les communard·e·s connu·e·s

#### Liste complète

```
SELECT ?communard_ou_communarde ?communard_ou_communardeLabel WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
}
```

In [4]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Fcommunard_ou_communarde%20%3Fcommunard_ou_communardeLabel%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>


#### Liste complète avec Reasonator

```
SELECT ?communard_ou_communarde ?communard_ou_communardeLabel
#pour récuprer l'adresse de la page reasonator en fr
(CONCAT("https://reasonator.toolforge.org/?q=", STRAFTER( str(?communard_ou_communarde) ,"http://www.wikidata.org/entity/"),"&lang=fr") AS ?reasonator) 
#pour récuprer l'adresse de la page reasonator en fr au format html
(CONCAT('<a href="','https://reasonator.toolforge.org/?q=', STRAFTER( str(?communard_ou_communarde) ,"http://www.wikidata.org/entity/"),'&lang=fr">', ?communard_ou_communardeLabel ,'</a>') AS ?reasonatorlien)
WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
}
```

In [2]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Fcommunard_ou_communarde%20%3Fcommunard_ou_communardeLabel%0A%23pour%20r%C3%A9cuprer%20l'adresse%20de%20la%20page%20reasonator%20en%20fr%0A(CONCAT(%22https%3A%2F%2Freasonator.toolforge.org%2F%3Fq%3D%22%2C%20STRAFTER(%20str(%3Fcommunard_ou_communarde)%20%2C%22http%3A%2F%2Fwww.wikidata.org%2Fentity%2F%22)%2C%22%26lang%3Dfr%22)%20AS%20%3Freasonator)%20%0A%23pour%20r%C3%A9cuprer%20l'adresse%20de%20la%20page%20reasonator%20en%20fr%20au%20format%20html%0A(CONCAT('%3Ca%20href%3D%22'%2C'https%3A%2F%2Freasonator.toolforge.org%2F%3Fq%3D'%2C%20STRAFTER(%20str(%3Fcommunard_ou_communarde)%20%2C%22http%3A%2F%2Fwww.wikidata.org%2Fentity%2F%22)%2C'%26lang%3Dfr%22%3E'%2C%20%3Fcommunard_ou_communardeLabel%20%2C'%3C%2Fa%3E')%20AS%20%3Freasonatorlien)%0AWHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%7D" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

#### Nombre
- 9 mars 2022 : 248
- 16 janvier 2023 : 297

Compteur actualisé en temps réel :

```
SELECT (Count(?communard_ou_communarde) as ?nombre_de_personne) WHERE {
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
}
group by ?nombre_de_personne
```

In [10]:
%%HTML
<iframe style="width: 65%; height: 20vh; border: none" 
src="https://query.wikidata.org/embed.html#SELECT%20(Count(%3Fcommunard_ou_communarde)%20as%20%3Fnombre_de_personne)%20WHERE%20%7B%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%7D%0Agroup%20by%20%3Fnombre_de_personne" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

#### Quelques tests possibles pour vérifier la fiabilité des données.

##### Pour vérifier qu'ils ne sont pas née après ou mort avant la Commune
```
SELECT ?communard ?communardLabel ?naissance?deces WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard wdt:P106 wd:Q1780490;
             wdt:P569 ?naissance;
             wdt:P570 ?deces.
   #FILTER(year(?deces)<1871 && year(?naissance)>1871) . 
}
```
-> au 16 janvier 2023 il y a entre 2 et 4 données à examiner correctement et corriger.

##### Des êtres vivants

```
SELECT  ?nature_de_l_élémentLabel (count(?nature_de_l_élément) as ?nb) WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
  OPTIONAL { ?communard_ou_communarde wdt:P31 ?nature_de_l_élément. }
}
Group by  ?nature_de_l_élément ?nature_de_l_élémentLabel
```

In [12]:
%%HTML
<iframe style="width:  65%; height: 25vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%20%3Fnature_de_l_%C3%A9l%C3%A9mentLabel%20(count(%3Fnature_de_l_%C3%A9l%C3%A9ment)%20as%20%3Fnb)%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%20%20OPTIONAL%20%7B%20%3Fcommunard_ou_communarde%20wdt%3AP31%20%3Fnature_de_l_%C3%A9l%C3%A9ment.%20%7D%0A%7D%0AGroup%20by%20%20%3Fnature_de_l_%C3%A9l%C3%A9ment%20%3Fnature_de_l_%C3%A9l%C3%A9mentLabel" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

Wikidata permet de rentrer pleins de monde, y compris des êtres de fictions. Il existe quelques communard-e-s de fictions dans par exemple le cri du peule, la série de BD communardes...

Au 16 janvier 2023 il y a 297 être humain (Q5) et 0 être humain de fiction (Q15632617).

> être humain (Q5) individu appartenant à l’espèce Homo sapiens, la seule espèce restante du genre Homo – distinct de « humain fictif » et de « humain possiblement fictif »

> humain de fiction (Q15632617)  	
être humain n'existant que dans des œuvres de fiction

La plupart des requête recherches les communard-e-s sans vérifier ce critère. Les réquetes seront à revoir (ou pas) quand un certains nombre de ces êtres fictionnels arriveront dans wikidata.


##### Quelques sources

Pour chaque personne, indique si possible :
- l'article wikipédia en français
- le numéro de sa page dans le maitron

> identifiant Maitron (P4724) identifiant d'une personne, d'un lieu de mémoire ou d'une monographie dans le Maitron en ligne, ensemble de dictionnaires biographiques du mouvement ouvrier

```
SELECT ?communard_ou_communarde ?communard_ou_communardeLabel ?article ?maiton WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.

  optional {?article schema:about ?communard_ou_communarde .
 ?article schema:isPartOf <https://fr.wikipedia.org/> .} 
  
  optional {?communard_ou_communarde wdt:P4724 ?maiton.}

}
```

In [12]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#%0ASELECT%20%3Fcommunard_ou_communarde%20%3Fcommunard_ou_communardeLabel%20%3Farticle%20%3Fmaiton%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%0A%20%20optional%20%7B%3Farticle%20schema%3Aabout%20%3Fcommunard_ou_communarde%20.%0A%20%3Farticle%20schema%3AisPartOf%20%3Chttps%3A%2F%2Ffr.wikipedia.org%2F%3E%20.%7D%20%0A%20%20%0A%20%20optional%20%7B%3Fcommunard_ou_communarde%20wdt%3AP4724%20%3Fmaiton.%7D%0A%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

On peut aussi compte combien de personnes ont un article dans wikipédia ou le maitron

```
SELECT (COUNT(?article) AS ?nb_WP) (COUNT(?maitron) AS ?nb_maitron) WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
  OPTIONAL {
    ?article schema:about ?communard_ou_communarde;
      schema:isPartOf <https://fr.wikipedia.org/>.
  }
  OPTIONAL { ?communard_ou_communarde wdt:P4724 ?maitron. }
}
```

In [13]:
%%HTML
<iframe style="width: 95%; height: 20vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20(COUNT(%3Farticle)%20AS%20%3Fnb_WP)%20(COUNT(%3Fmaitron)%20AS%20%3Fnb_maitron)%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%20%20OPTIONAL%20%7B%0A%20%20%20%20%3Farticle%20schema%3Aabout%20%3Fcommunard_ou_communarde%3B%0A%20%20%20%20%20%20schema%3AisPartOf%20%3Chttps%3A%2F%2Ffr.wikipedia.org%2F%3E.%0A%20%20%7D%0A%20%20OPTIONAL%20%7B%20%3Fcommunard_ou_communarde%20wdt%3AP4724%20%3Fmaitron.%20%7D%0A%7D" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

### Présentations et statistiques à propos des Communard-e-s

L'intérêt de wikidata est de pouvoir faire quelques statistiques et autres affichages.

#### Galerie de portrait
Tous ces portrait sont hébergé sur wikicommons et sont sous une licence libre (variable d'une image à l'autre) et donc réutilisable .

```
#defaultView:ImageGrid
SELECT DISTINCT ?communard_ou_communarde ?communard_ou_communardeLabel ?image WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
  OPTIONAL { ?communard_ou_communarde wdt:P18 ?image. }
}
```

In [14]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#%23defaultView%3AImageGrid%0ASELECT%20DISTINCT%20%3Fcommunard_ou_communarde%20%3Fcommunard_ou_communardeLabel%20%3Fimage%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%20%20OPTIONAL%20%7B%20%3Fcommunard_ou_communarde%20wdt%3AP18%20%3Fimage.%20%7D%0A%7D" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

#### Gender-gap 
Ou en bon français, la répartition homme/femme.

On mesure ici le nombre d'hommes et de femme référencé dans wikidata. On mesure donc le ratio entre les personnes rentrées dans wikidata et pas dans la réalité. Même si les critères d’admissibilité dans wikidata sont très généreux, il faut un minimum de sources.

> sexe ou genre (P21) pour les humains, soit masculin (Q6581097), féminin (Q6581072), intersexué (Q1097630), femme transgenre (Q1052281), homme transgenre (Q2449503), non-binaire (Q48270) ; pour les animaux, mâle (Q44148) ou femelle (Q43445)

```
#defaultView:BubbleChart
SELECT ?sexe_ou_genreLabel (count(?sexe_ou_genre) as ?genre) WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490.
  OPTIONAL { ?communard_ou_communarde wdt:P21 ?sexe_ou_genre. }
}
group by ?sexe_ou_genreLabel
```

In [15]:
%%HTML
<iframe style="width: 95%; height: 40vh; border: none;" 
src="https://query.wikidata.org/embed.html#%23defaultView%3ABubbleChart%0ASELECT%20%3Fsexe_ou_genreLabel%20(count(%3Fsexe_ou_genre)%20as%20%3Fgenre)%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490.%0A%20%20OPTIONAL%20%7B%20%3Fcommunard_ou_communarde%20wdt%3AP21%20%3Fsexe_ou_genre.%20%7D%0A%7D%0Agroup%20by%20%3Fsexe_ou_genreLabel" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

- 9 mars 2022 : 172 / 74 soit 2.32
- 16 janvier 2023 : 190 / 107 soit 1.77

#### A part avoir participer à la commune, que faisaient ces personnes ?

la propriété P106, déjà évoqué, permet de connaitre leur occupation : métiers, loisir...

On peut compter combien de personne participe à chaque occupation. Une personen peut avoir de multiple occupation. Elles peuvent avoir été occupée avant/pendant/après les événements.

##### Au moins deux personnes qui l'occupe

```
SELECT ?occupation ?occupationLabel (COUNT(?communard_ou_communarde) AS ?nombre) WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490;
                           wdt:P106 ?occupation.
}
GROUP BY ?occupation ?occupationLabel
HAVING (((COUNT(?communard_ou_communarde)) > 1 ) && ((COUNT(?communard_ou_communarde)) < 290 ))# Sélectionner ceux dont il y a plus d'une occurence et moins de 290 pour supprimer communard qui est à plus de 297 le 12 janvier 2023

ORDER BY DESC (?nombre)
```

In [13]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Foccupation%20%3FoccupationLabel%20(COUNT(%3Fcommunard_ou_communarde)%20AS%20%3Fnombre)%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP106%20%3Foccupation.%0A%7D%0AGROUP%20BY%20%3Foccupation%20%3FoccupationLabel%0AHAVING%20(((COUNT(%3Fcommunard_ou_communarde))%20%3E%201%20)%20%26%26%20((COUNT(%3Fcommunard_ou_communarde))%20%3C%20290%20))%23%20S%C3%A9lectionner%20ceux%20dont%20il%20y%20a%20plus%20d'une%20occurence%20et%20moins%20de%20290%20pour%20supprimer%20communard%20qui%20est%20%C3%A0%20plus%20de%20297%20le%2012%20janvier%202023%0A%0AORDER%20BY%20DESC%20(%3Fnombre)" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

#### Quel age avait ces personnes en 1871

```
#defaultView:AreaChart
SELECT 
  ?age_on_march_18th (COUNT(?communard) AS ?count_persons)
WHERE {
  ?communard wdt:P106 wd:Q1780490;
             wdt:P569 ?birth_date.
  BIND (YEAR("1871-03-18"^^xsd:date) - YEAR(?birth_date) AS ?age_on_march_18th)
  FILTER(?age_on_march_18th > 5)
}
GROUP BY ?age_on_march_18th
```

In [17]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#%23defaultView%3AAreaChart%0ASELECT%20%0A%20%20%3Fage_on_march_18th%20(COUNT(%3Fcommunard)%20AS%20%3Fcount_persons)%0AWHERE%20%7B%0A%20%20%3Fcommunard%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP569%20%3Fbirth_date.%0A%20%20BIND%20(YEAR(%221871-03-18%22%5E%5Exsd%3Adate)%20-%20YEAR(%3Fbirth_date)%20AS%20%3Fage_on_march_18th)%0A%20%20FILTER(%3Fage_on_march_18th%20%3E%205)%0A%7D%0AGROUP%20BY%20%3Fage_on_march_18th%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>


### Relations entre communard·e·s

#### Généalogie
communard dont les enfants sont communards
```
SELECT ?communard ?communardLabel ?enfant ?enfantLabel
WHERE {
SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }

  ?communard wdt:P106 wd:Q1780490;
              wdt:P40 ?enfant.
  ?enfant wdt:P106 wd:Q1780490.
}
```

grand parent -> petits enfants
```
SELECT ?grand_parent ?grand_parentLabel ?parent ?parentLabel ?communard ?communardLabel ?enfant ?enfantLabel ?petit_enfant ?petit_enfantLabel
WHERE {
SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }

  ?communard wdt:P106 wd:Q1780490.
  #Optional {?parent wdt:P40 ?communard.}
  #Optional {?grand_parent wdt:P40 ?parent.}
  Optional {?communard wdt:P40 ?enfant.}
  #Optional {?enfant wdt:P40 ?petit_enfant.}
}
```

https://www.entitree.com/en/family_tree/Herminie_Cadolle?0d0=d
-> petit fils champion cycliste

https://www.entitree.com/fr/arbre_g%C3%A9n%C3%A9alogique/Charles_Longuet?0d0=d&0d0d1=dr&0d1=dr&0d2=d&0d1d2=d

#### Reseau social des communard·e·s sur wikidata

##### Graph des communard·e·s et des organismes dont als sont membres

membre de P463

```
#defaultView:Graph
SELECT ?communard_ou_communarde ?communard_ou_communardeLabel ?membre_de ?membre_deLabel WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490;
                           wdt:P463 ?membre_de.
}
````

In [18]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#%23defaultView%3AGraph%0ASELECT%20%3Fcommunard_ou_communarde%20%3Fcommunard_ou_communardeLabel%20%3Fmembre_de%20%3Fmembre_deLabel%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP463%20%3Fmembre_de.%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

##### nombre par organisme

```
SELECT ?membre_de ?membre_deLabel (count(?membre_deLabel) as ?nombre_de_membre) WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?communard_ou_communarde wdt:P106 wd:Q1780490;
                           wdt:P463 ?membre_de.
  optional {?membre_de wdt:P18 ?image.}
}

group by ?membre_de ?membre_deLabel
order by desc(?nombre_de_membre)
```

In [14]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Fmembre_de%20%3Fmembre_deLabel%20(count(%3Fmembre_deLabel)%20as%20%3Fnombre_de_membre)%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP463%20%3Fmembre_de.%0A%20%20optional%20%7B%3Fmembre_de%20wdt%3AP18%20%3Fimage.%7D%0A%7D%0A%0Agroup%20by%20%3Fmembre_de%20%3Fmembre_deLabel%0Aorder%20by%20desc(%3Fnombre_de_membre)" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

## Les communardes et communards dans des cimetières
### Généralités
Les communards étant touz* décédæ, c'est notamment dans les cimetières, quand les tombes existent, qu'on peut leur rendre hommage. N'oublions pas que beaucoup sont dans des fossés communes anonymes.

En termes de méthode wikidata nous propose la propriété lieu de sépulture (P119). Selon les précisions cela peut être une ville ou un cimetière. Cet élément possède une propriété coordonnées géographiques.

Pour certaines personnes, la propriété photo de la sépulture est rempli avec une photo de communs.
Pour certaines personnes il existe même un élément pour leur leur tombe. 

On peut donc connaître le nombre de personnes par cimetière.
On peut même les repérer sur un plan.

Au 14 janvier seul le cimetière du Montparnasse est "complet". Celui du père Lachaise est quasiment finit. Aucun des deux ne prends en compte les monuments collectifs.

Pour certain·e·s communard·e·s, le lieu de sépulture (P119) est connu. Pour certain·e·s les coordonnées géographiques de la tombe sont dans wikidata.


> lieu de sépulture (P119)
lieu où le corps du sujet est conservé

### Nombre de personne par cimetière

```
SELECT ?lieu_de_sépulture ?lieu_de_sépultureLabel ?coordonnees (COUNT(*) AS ?count) WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
 ?communard_ou_communarde wdt:P106 wd:Q1780490;
                           wdt:P119 ?lieu_de_sépulture.
 ?lieu_de_sépulture wdt:P31/wdt:P279* wd:Q39614;
                    wdt:P625 ?coordonnees.
}

group by ?lieu_de_sépulture ?lieu_de_sépultureLabel ?coordonnees
order by desc (?count)
```

In [20]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Flieu_de_s%C3%A9pulture%20%3Flieu_de_s%C3%A9pultureLabel%20%3Fcoordonnees%20(COUNT(*)%20AS%20%3Fcount)%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%3Fcommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP119%20%3Flieu_de_s%C3%A9pulture.%0A%20%3Flieu_de_s%C3%A9pulture%20wdt%3AP31%2Fwdt%3AP279*%20wd%3AQ39614%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP625%20%3Fcoordonnees.%0A%7D%0A%0Agroup%20by%20%3Flieu_de_s%C3%A9pulture%20%3Flieu_de_s%C3%A9pultureLabel%20%3Fcoordonnees%0Aorder%20by%20desc%20(%3Fcount)" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

### Cimetière du Montparnasse
Cartes des tombes de communard·e·s ou de personnes ayant un lien avec la Commune dans le cimetière du Montparnasse (Q272208).


```
#defaultView:Map
SELECT Distinct ?tombe ?tombeLabel ?image_Tombe ?coordonnées ?tombe_deLabel ?image_personne WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  
  #Qui, les communard-e-s et la liste des personnes lors de la cérémonie
  ?communard wdt:P106 wd:Q1780490;
             wdt:P119 wd:Q272208.
  Values ?personnalite {wd:Q313709 
                        wd:Q61897268 
                        wd:Q5749
                        wd:Q1959369
                        wd:Q3047711
                        wd:Q3022660
                        wd:Q551648
                        wd:Q704649
                        wd:Q2826417
                        wd:Q3302715
                        wd:Q337628}
  #Chercher les tombes du cimetière montparnasse                      
  ?tombe wdt:P361 wd:Q272208;
         wdt:P547 ?tombe_de.        
  #Prendre les tombes voulues
  filter (?tombe_de =?communard || ?tombe_de=?personnalite)
  
  OPTIONAL {?tombe wdt:P18 ?image_Tombe.}
  OPTIONAL {?tombe wdt:P625 ?coordonnées.}
  OPTIONAL {?tombe_de wdt:P18 ?image_personne.}
  
}
```

In [15]:
%%HTML
<iframe style="width: 95%; height: 75vh; border: none;" 
src="https://query.wikidata.org/embed.html#%23defaultView%3AMap%0ASELECT%20Distinct%20%3Ftombe%20%3FtombeLabel%20%3Fimage_Tombe%20%3Fcoordonn%C3%A9es%20%3Ftombe_deLabel%20%3Fimage_personne%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%0A%20%20%23Qui%2C%20les%20communard-e-s%20et%20la%20liste%20des%20personnes%20lors%20de%20la%20c%C3%A9r%C3%A9monie%0A%20%20%3Fcommunard%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP119%20wd%3AQ272208.%0A%20%20Values%20%3Fpersonnalite%20%7Bwd%3AQ313709%20%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ61897268%20%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ5749%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ1959369%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ3047711%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ3022660%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ551648%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ704649%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ2826417%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ3302715%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wd%3AQ337628%7D%0A%20%20%23Chercher%20les%20tombes%20du%20cimeti%C3%A8re%20montparnasse%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%0A%20%20%3Ftombe%20wdt%3AP361%20wd%3AQ272208%3B%0A%20%20%20%20%20%20%20%20%20wdt%3AP547%20%3Ftombe_de.%20%20%20%20%20%20%20%20%0A%20%20%23Prendre%20les%20tombes%20voulues%0A%20%20filter%20(%3Ftombe_de%20%3D%3Fcommunard%20%7C%7C%20%3Ftombe_de%3D%3Fpersonnalite)%0A%20%20%0A%20%20OPTIONAL%20%7B%3Ftombe%20wdt%3AP18%20%3Fimage_Tombe.%7D%0A%20%20OPTIONAL%20%7B%3Ftombe%20wdt%3AP625%20%3Fcoordonn%C3%A9es.%7D%0A%20%20OPTIONAL%20%7B%3Ftombe_de%20wdt%3AP18%20%3Fimage_personne.%7D%0A%20%20%0A%7D" 
referrerpolicy="origin" 
sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

### Cimetière du Père la Chaise
Cartes des tombes de communard·e·s ou de personnes ayant un lien avec la Commune dans le cimetière du Père-Lahaise (Q311).

```
#defaultView:Map
SELECT Distinct ?tombe ?tombeLabel ?image_Tombe ?coordonnées ?tombe_deLabel ?image_personne WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  
  #Qui, les communard-e-s
  ?communard wdt:P106 wd:Q1780490;
             wdt:P119 wd:Q311.

  #Chercher les tombes du cimetière montparnasse                      
  ?tombe wdt:P276 ?lieu;
         wdt:P547 ?tombe_de. 
  
  ?lieu wdt:P361 wd:Q311.
        
  #Prendre les tombes voulues
  filter (?tombe_de =?communard)
  
  OPTIONAL {?tombe wdt:P18 ?image_Tombe.}
  OPTIONAL {?tombe wdt:P625 ?coordonnées.}
  OPTIONAL {?tombe_de wdt:P18 ?image_personne.}
  
}
```

In [16]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#%23defaultView%3AMap%0ASELECT%20Distinct%20%3Ftombe%20%3FtombeLabel%20%3Fimage_Tombe%20%3Fcoordonn%C3%A9es%20%3Ftombe_deLabel%20%3Fimage_personne%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%0A%20%20%23Qui%2C%20les%20communard-e-s%0A%20%20%3Fcommunard%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP119%20wd%3AQ311.%0A%0A%20%20%23Chercher%20les%20tombes%20du%20cimeti%C3%A8re%20montparnasse%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%0A%20%20%3Ftombe%20wdt%3AP276%20%3Flieu%3B%0A%20%20%20%20%20%20%20%20%20wdt%3AP547%20%3Ftombe_de.%20%0A%20%20%0A%20%20%3Flieu%20wdt%3AP361%20wd%3AQ311.%0A%20%20%20%20%20%20%20%20%0A%20%20%23Prendre%20les%20tombes%20voulues%0A%20%20filter%20(%3Ftombe_de%20%3D%3Fcommunard)%0A%20%20%0A%20%20OPTIONAL%20%7B%3Ftombe%20wdt%3AP18%20%3Fimage_Tombe.%7D%0A%20%20OPTIONAL%20%7B%3Ftombe%20wdt%3AP625%20%3Fcoordonn%C3%A9es.%7D%0A%20%20OPTIONAL%20%7B%3Ftombe_de%20wdt%3AP18%20%3Fimage_personne.%7D%0A%20%20%0A%7D%0A"referrerpolicy="origin" 
sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>


## Une chronologie de la Commune

Pour tous les éléments faisant partie de (P361) la Commune, affiche leurs dates connues.
Idem pour les communard·e·s.

De ce tableau ou peut faire une frise chronologique.

```
SELECT 
?Commune_de_Paris ?Commune_de_ParisLabel ?date ?date_de_début ?date_de_fin
?Communard_ou_communarde ?Communard_ou_communardeLabel ?date_deces
WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  {
    # Partie de la Commune
    ?Commune_de_Paris wdt:P361 wd:Q133132.
  OPTIONAL { ?Commune_de_Paris wdt:P585 ?date. }
  OPTIONAL { ?Commune_de_Paris wdt:P580 ?date_de_début. }
  OPTIONAL { ?Commune_de_Paris wdt:P582 ?date_de_fin. }
    }
  Union
  {
  # Communard-e-s
  ?Communard_ou_communarde wdt:P106 wd:Q1780490;
                           wdt:P570 ?date_deces.
    FILTER (year(?date_deces)<1872)
    }
}
```

In [83]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%0A%3FCommune_de_Paris%20%3FCommune_de_ParisLabel%20%3Fdate%20%3Fdate_de_d%C3%A9but%20%3Fdate_de_fin%0A%3FCommunard_ou_communarde%20%3FCommunard_ou_communardeLabel%20%3Fdate_deces%0AWHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%23%20Partie%20de%20la%20Commune%0A%20%20%7B%0A%20%20%20%20%3FCommune_de_Paris%20wdt%3AP361%20wd%3AQ133132.%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP585%20%3Fdate.%20%7D%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP580%20%3Fdate_de_d%C3%A9but.%20%7D%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP582%20%3Fdate_de_fin.%20%7D%0A%20%20%20%20%7D%0A%20%20Union%0A%20%20%7B%0A%20%20%23%20Communard-e-s%0A%20%20%3FCommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP570%20%3Fdate_deces.%0A%20%20%20%20FILTER%20(year(%3Fdate_deces)%3C1872)%0A%20%20%20%20%7D%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

### version étendue
avec commune, semaine sanglante, campagne de l'intérieur

```
SELECT distinct
?Commune_de_Paris ?Commune_de_ParisLabel ?date ?date_de_début ?date_de_fin ?date_de_publication
?Communard_ou_communarde ?Communard_ou_communardeLabel ?date_deces
WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  values ?periodes {wd:Q133132 #commune de Paris
          wd:Q3478362 #semaine sanglante
          wd:Q47007501 #campagne de l'intérieur
          }
          
  {
    # Partie de la Commune
    ?Commune_de_Paris wdt:P361 ?periodes
  OPTIONAL { ?Commune_de_Paris wdt:P585 ?date. }
  OPTIONAL { ?Commune_de_Paris wdt:P580 ?date_de_début. }
  OPTIONAL { ?Commune_de_Paris wdt:P582 ?date_de_fin. }
  OPTIONAL { ?Commune_de_Paris wdt:P577 ?date_de_publication.}
    }
  Union
  {
  # Communard-e-s
  ?Communard_ou_communarde wdt:P106 wd:Q1780490;
                           wdt:P570 ?date_deces.
    FILTER (year(?date_deces)<1872)
    }
}
```

In [23]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20distinct%0A%3FCommune_de_Paris%20%3FCommune_de_ParisLabel%20%3Fdate%20%3Fdate_de_d%C3%A9but%20%3Fdate_de_fin%20%3Fdate_de_publication%0A%3FCommunard_ou_communarde%20%3FCommunard_ou_communardeLabel%20%3Fdate_deces%0AWHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20values%20%3Fperiodes%20%7Bwd%3AQ133132%20%23commune%20de%20Paris%0A%20%20%20%20%20%20%20%20%20%20wd%3AQ3478362%20%23semaine%20sanglante%0A%20%20%20%20%20%20%20%20%20%20wd%3AQ47007501%20%23campagne%20de%20l'int%C3%A9rieur%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20%0A%20%20%7B%0A%20%20%20%20%23%20Partie%20de%20la%20Commune%0A%20%20%20%20%3FCommune_de_Paris%20wdt%3AP361%20%3Fperiodes%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP585%20%3Fdate.%20%7D%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP580%20%3Fdate_de_d%C3%A9but.%20%7D%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP582%20%3Fdate_de_fin.%20%7D%0A%20%20OPTIONAL%20%7B%20%3FCommune_de_Paris%20wdt%3AP577%20%3Fdate_de_publication.%7D%0A%20%20%20%20%7D%0A%20%20Union%0A%20%20%7B%0A%20%20%23%20Communard-e-s%0A%20%20%3FCommunard_ou_communarde%20wdt%3AP106%20wd%3AQ1780490%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wdt%3AP570%20%3Fdate_deces.%0A%20%20%20%20FILTER%20(year(%3Fdate_deces)%3C1872)%0A%20%20%20%20%7D%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

## Les affiches

```
SELECT ?affiche ?afficheLabel ?date_de_publication ?image WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?affiche wdt:P31 wd:Q429785.
  {?affiche wdt:P361 wd:Q690489.}
  Union {?affiche wdt:P361 wd:Q133132}
  OPTIONAL { ?affiche wdt:P577 ?date_de_publication. }
  OPTIONAL { ?affiche wdt:P18 ?image. }
}
```

In [23]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Faffiche%20%3FafficheLabel%20%3Fdate_de_publication%20%3Fimage%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Faffiche%20wdt%3AP31%20wd%3AQ429785.%0A%20%20%7B%3Faffiche%20wdt%3AP361%20wd%3AQ690489.%7D%0A%20%20Union%20%7B%3Faffiche%20wdt%3AP361%20wd%3AQ133132%7D%0A%20%20OPTIONAL%20%7B%20%3Faffiche%20wdt%3AP577%20%3Fdate_de_publication.%20%7D%0A%20%20OPTIONAL%20%7B%20%3Faffiche%20wdt%3AP18%20%3Fimage.%20%7D%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

## Les barricades

```
SELECT ?barricade ?barricadeLabel ?coordonnées_géographiques ?image ?catégorie_Commons ?date_de_début ?date_de_fin WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?barricade wdt:P31 wd:Q81715;
    wdt:P361 wd:Q133132.
  OPTIONAL { ?barricade wdt:P625 ?coordonnées_géographiques. }
  OPTIONAL { ?barricade wdt:P18 ?image. }
  OPTIONAL { ?barricade wdt:P373 ?catégorie_Commons. }
  OPTIONAL { ?barricade wdt:P580 ?date_de_début. }
  OPTIONAL { ?barricade wdt:P582 ?date_de_fin. }
}
```

In [17]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3Fbarricade%20%3FbarricadeLabel%20%3Fcoordonn%C3%A9es_g%C3%A9ographiques%20%3Fimage%20%3Fcat%C3%A9gorie_Commons%20%3Fdate_de_d%C3%A9but%20%3Fdate_de_fin%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3Fbarricade%20wdt%3AP31%20wd%3AQ81715%3B%0A%20%20%20%20wdt%3AP361%20wd%3AQ133132.%0A%20%20OPTIONAL%20%7B%20%3Fbarricade%20wdt%3AP625%20%3Fcoordonn%C3%A9es_g%C3%A9ographiques.%20%7D%0A%20%20OPTIONAL%20%7B%20%3Fbarricade%20wdt%3AP18%20%3Fimage.%20%7D%0A%20%20OPTIONAL%20%7B%20%3Fbarricade%20wdt%3AP373%20%3Fcat%C3%A9gorie_Commons.%20%7D%0A%20%20OPTIONAL%20%7B%20%3Fbarricade%20wdt%3AP580%20%3Fdate_de_d%C3%A9but.%20%7D%0A%20%20OPTIONAL%20%7B%20%3Fbarricade%20wdt%3AP582%20%3Fdate_de_fin.%20%7D%0A%7D%0A" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>

## Elle n'est pas morte

>  commémore (P547)
ce que commémore le lieu, le monument, le mémorial, le jour férié

```
SELECT ?Commune_de_Paris ?Commune_de_ParisLabel WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  ?Commune_de_Paris wdt:P547 wd:Q133132.
}
LIMIT 100
```

In [25]:
%%HTML
<iframe style="width: 95%; height: 50vh; border: none;" 
src="https://query.wikidata.org/embed.html#SELECT%20%3FCommune_de_Paris%20%3FCommune_de_ParisLabel%20WHERE%20%7B%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%20%20%3FCommune_de_Paris%20wdt%3AP547%20wd%3AQ133132.%0A%7D%0ALIMIT%20100" 
referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups">
</iframe>